content/doc/book/using/best-practices.adoc

```diff
@@ -0,0 +1,143 @@
+---
+layout: section
+title: Best Practices
+---
+ifdef::backend-html5[]
+:description:
+:author:
+:sectanchors:
+:toc:
+:toclevels: 4
+:hide-uri-scheme:
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
+endif::[]
+
+Continuous Integration (CI) with automated test execution and trends has changed the way companies look at build management, release management, deployment automation, and test orchestration.
+This section describes best practices to provide executives, business managers, software developers, and architects with a better sense of what Jenkins can contribute throughout the project lifecycle.
+
+== Use organization folders
+
+Organization Folders offer a convenient way to allow Jenkins to automatically create and delete jobs as repositories and branches are created and deleted.
+Particularly, if you use GitHub organizations, Bitbucket teams, GitLab groups, or Gitea organizations, then Jenkins can automatically detect the creation of a new repository and create a Multibranch Pipeline project for it.
+
+See the link:/doc/book/pipeline-as-code/[organization folders] documentation for more details.
+
+.GitHub organization folders
+video::LbXKUKQ24T8[youtube,width=800,height=420]
+
+.GitLab group folders
+video::it6TOeQ6EHg[youtube,width=800,height=420]
+
+.Bitbucket project folders
+video::85b6fiVolfk[youtube,width=800,height=420]
+
+.Gitea organization folders
+video::NO3sZWRxgQM[youtube,width=800,height=420]
+
+== Use multibranch Pipelines
+
+If something prevents you from using organization folders, then use multibranch Pipelines.
+
+Insert more wisdom here ...
+
+See the link:/doc/book/pipeline-as-code/[multibranch Pipelines] documentation for more details.
+
+== Use Pipeline
+
+Insert Pipeline wisdom here
+
+See the link:/doc/book/pipeline/[Pipeline] documentation for more details.
+
+video::IOUm1lw7F58[youtube,width=800,height=420]
+
+== Use reports effectively
+
+Charts and graphs show trends and patterns that can highlight project status and progress.
+Automated test results for unit tests, integration tests, and end to end tests can highlight brittleness or instability.
+Coverage reports can illustrate areas that automated tests are not executing.
+Compiler warning messages can be a first indication of a problem.
+Static analysis tools can report risky code or code that may have security risks.
+Performance test results can idenitfy delays or problem areas.
+
+The plugin:warnings-ng[warnings next generation] plugin provides convenient access to many reports, including:
+
+* compiler warnnings and errors (like gcc, clang, javac, golang, etc.)
+* static analysis warnings and errors (like spotbugs, checkstyle, pmd, lint, cpd, Simian, etc.)
+* code coverage reports
+
+video::tj3xYFA6Q2o[youtube,width=800,height=420]
```

Comment:
```suggestion
The plugin:warnings-ng[Warnings Next Generation] plugin provides convenient access to many reports, including:

* Compiler warnnings and errors (like gcc, clang, javac, or  golang)
* Static analysis warnings and errors (spotbugs, checkstyle, pmd, lint, cpd, or Simian)
* Code coverage reports

.How to use the Warnings Next Generation plugin 
video::tj3xYFA6Q2o[youtube,width=800,height=420]
```
suggestion to use the capitalized name for the plugin, and adjusting the text for some standardization/capitalization